### PR TITLE
Replace Select2

### PR DESCRIPTION
**Pull request by swaterkamp**
The old Select2 component is replaced by Select and MultiSelect respectively. The two new components are native React components and they replace the old jQuery version.

### CODE REVIEW

**Comment by bjoernricks** [comment]
Next step would be to replace the options by settings the items prob. Most of the time this can be done by replacing 
```jsx
const xyz_opts = render_options(...)
<select>{xyz_opts}</select>
```
with
```jsx
const xyz_items = render_select_items(...)
<select items={xyz_items} />
```

File: ng/src/web/components/form/select.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ class Select extends React.Component {
       className,
       disabled = false,
       items,
-      menuPosition,
+      menuPosition = 'adjust',
```
Comment (bjoernricks) [inline ng/src/web/components/form/select.js]: what's the reason for the change? i don't think that makes sense. it breaks e.g. the powerfilter

File: ng/src/web/components/form/multiselect.js [hunk 2]
```diff
@@ -162,7 +164,8 @@ class MultiSelect extends React.Component {
       className,
       disabled = false,
       items,
-      menuPosition,
+      menuPosition = 'adjust',
```
Comment (bjoernricks) [inline ng/src/web/components/form/multiselect.js]: Please adjust your commit message of f1f6e9ae5d982a50475d1bda2cd8fdc3a247b546 or split the commit into to separated commits. Your are not mentioning the setting a new default for menuPosition.

File: ng/src/web/components/form/select.js (hunk 1 above)
Comment (swaterkamp) [reply ng/src/web/components/form/select.js]: For the MultiSelect it made sense to have a menu position that looks decent by default. I thought it to be a good idea to have that for Select as well. I didn't recognize this breaks the powerfilter.
